### PR TITLE
A few small fixes

### PR DESCRIPTION
**Pull request by hvanz**
- [Makefile: use runner commit hash for configgen](https://github.com/cometbft/qa-infra/commit/2bfaf13a2a5bd0eb109f96a9682c556802a34172)
- [Remove unused ansible files](https://github.com/cometbft/qa-infra/commit/c8befbbc96b7d24c72b2974a42c327cb685b1f04)
- [Fix websocket endpoints](https://github.com/cometbft/qa-infra/commit/42b5b874064b8aab288d4140e73980b756b2b961)
- [configgen.sh: download latencies file + change config to allow big txs via websocket](https://github.com/cometbft/qa-infra/commit/ff854aba7e8545a82c786960a710e59f13a52bda)
- [Makefile: remove unused target ansible-install-retry](https://github.com/cometbft/qa-infra/pull/39/commits/1dcde9b28b04738fd6014da137313395d44392b1)
- [loadrunners-init.yaml: fix file permissions](https://github.com/cometbft/qa-infra/pull/39/commits/bd31f20c8c8881ee3e9415cc33105a7f501df14a)
- [runload.yaml: use first validator as default endpoint and fix endpoint port](https://github.com/cometbft/qa-infra/pull/39/commits/d2a6c43485496083c1d0ccab195a54e8a0886f39)

### CODE REVIEW

File: ansible/loadrunners-init.yaml [hunk 1]
```diff
@@ -3,23 +3,25 @@
...
-    - name: Copy script files
-      ansible.builtin.copy:
-        src: "{{ item.src }}"
-        dest: /root
-      loop:
-        - src: scripts/200-node-loadscript.sh
-        - src: scripts/rotating-loadscript.sh
-      become: true
-      mode: preserve
+  - name: Check if load tool exists
```
Comment (mzabaluev) [inline ansible/loadrunners-init.yaml:6]: I was going to complain about the changed indentation, but learned that it's the recommended way.

File: ansible/runload.yaml [hunk 2]
```diff
@@ -4,19 +4,20 @@
   hosts: loadrunners
   vars:
     ansible_host_key_checking: false
-    endpoints: ws://127.0.0.1:26656/websocket # comma separated list of endpoints
+    validator_0_ip: "{{ groups['validators'][0] }}"
+    validator_0_internal_ip: "{{ hostvars[validator_0_ip].internal_ip }}"
```
Comment (mzabaluev) [inline ansible/runload.yaml:8]: Why should this expression have to go through `hostvars` to get at `validator_0_ip`...

File: ansible/runload.yaml [hunk 3]
```diff
@@ -4,19 +4,20 @@
...
+    endpoints: ws://{{ validator_0_internal_ip }}:26657/v1/websocket # comma separated list of endpoints
```
Comment (mzabaluev) [inline ansible/runload.yaml:9]: ... while this one can apparently use the `validator_0_internal_ip` variable directly?

File: script/configgen.sh [hunk 4]
```diff
@@ -71,13 +71,21 @@ ifd-from-ansible $HOSTS_PATH $IFD_PATH $VPC_SUBNET
...
 
 
 for file in `find ./ansible/testnet/ -name config.toml -type f`; do
 	sed $INPLACE_SED_FLAG "s/unsafe = .*/unsafe = true/g" $file
 	sed $INPLACE_SED_FLAG "s/prometheus = .*/prometheus = true/g" $file
-	sed $INPLACE_SED_FLAG "s/cache_size = .*/cache_size = 200000/g" $file
+
+	# to allow sending big txs via websockets
+	sed $INPLACE_SED_FLAG "s/max_body_bytes = .*/max_body_bytes = 2097152/g" $file
+	sed $INPLACE_SED_FLAG "s/max_header_bytes = .*/max_header_bytes = 2097152/g" $file
```
Comment (mzabaluev) [inline script/configgen.sh:88]: Curious if the `g` flag does anything here: it does not work across multiple lines (also the regex certainly is not meant to be multiline), and there's only one statement per line that you can meaningfully edit.

File: ansible/loadrunners-init.yaml (hunk 1 above)
Comment (hvanz) [reply ansible/loadrunners-init.yaml]: Ok, I changed it back also to keep it consistent with the other playbooks.

File: ansible/runload.yaml (hunk 2 above)
Comment (hvanz) [reply ansible/runload.yaml]: Right. I removed the two variable definitions and now just access directly the value I want in `hostvars`.

File: ansible/runload.yaml (hunk 3 above)
Comment (hvanz) [reply ansible/runload.yaml]: (see comment above)

File: script/configgen.sh (hunk 4 above)
Comment (hvanz) [reply script/configgen.sh]: Yes, the expressions appear only once in the file, so `g` is not needed. I removed it.

File: ansible/runload.yaml [hunk 5]
```diff
@@ -4,9 +4,7 @@
   hosts: loadrunners
   vars:
     ansible_host_key_checking: false
-    validator_0_ip: "{{ groups['validators'][0] }}"
-    validator_0_internal_ip: "{{ hostvars[validator_0_ip].internal_ip }}"
-    endpoints: ws://{{ validator_0_internal_ip }}:26657/v1/websocket # comma separated list of endpoints
+    endpoints: ws://{{ hostvars[groups[validators][0]].internal_ip }}:26657/v1/websocket # comma separated list of endpoints
```
Comment (mzabaluev) [inline ansible/runload.yaml:7]: I don't actually mind assigning some variables for readability, just the way it was done looked a bit inconsistent. It was a small nitpick, anyway.

**Review by mzabaluev — APPROVED** [review]
LGTM

**Pull request merged**